CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 
 ##### Enhancements
 
+* Add `implicitly_unwrapped_optional` rule
+  that warns when using implicitly unwrapped optional,
+  except cases when this IUO is IBOutlet.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes